docs/getting-started/os-support.rst

```diff
@@ -1,11 +1,16 @@
 OS Support by Platform and Version
 ==================================
 
-The following matrix shows which Operating Systems, Platforms, and Containers / Instance Engines are supported with which versions of Scylla.
+The following matrix shows which Linux Distribution, Containers, and images are supported with which versions of ScyllaDB.
 
-Scylla requires a fix to the XFS append introduced in kernel 3.15 (back-ported to 3.10 in RHEL/CentOS).
-Scylla will not run with earlier kernel versions. Details in `Scylla issue 885 <https://github.com/scylladb/scylla/issues/885>`_.
```

Comment:
Are these two statements which you removed here no longer relevant, because none of the "supported" versions have this problem? Or what?

Reply:
This is following the comment in the original issue

> The second paragraph of the page says "Scylla requires a fix to the XFS append introduced in kernel 3.15 (back-ported to 3.10 in RHEL/CentOS). Scylla will not run with earlier kernel versions. Details in https://github.com/scylladb/scylla/issues/885.". But Linux 3.15 was released 5 years ago. If this comment is relevant to anything, it's only relevant to some very old distributions mentioned in the table below (like CentOS 7). I think it belongs as a footnote to some of the old distributions listed, not as part of the introduction.